saleor/graphql/order/filters.py

```diff
@@ -201,6 +207,7 @@ class OrderFilter(DraftOrderFilter):
     numbers = ListObjectTypeFilter(
         input_class=graphene.String, method=filter_by_order_number
     )
+    checkout_token = django_filters.CharFilter(method=filter_checkouts)
```

Comment:
Did you consider making it a list filter that can return multiple results when multiple tokens are passed? We have the same for `ids` and `numbers`.

Reply:
Yeah I was actually wondering about that. But then I believe it would make sense to add `checkout_token` to the `order` type. And I'm not sure about that one.

Reply:
Yep, that would make sense. I think we could do that but maybe we should discuss. @fowczarek @korycins What do you think?

Reply:
Yea we could show this field to Order Managers. It's could be helpful in debugging payment errors. 
I think we should filter by multiple tokens like in other places. 
I also consider that we should use `CheckoutID's` instead of tokens. What do you think? As far as I remember we deprecate fetching checkout by tokens and we require ids instead.

Reply:
I also wondered about using ID's instead of tokens and I'm fine with it. @mmiszy What do you think, will it be still good for you guys?

Reply:
As long as I'll be able to query orders by checkout id - it's fine :)